nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/FXMLController.java

```diff
@@ -863,6 +861,39 @@ public void initialize(URL url, ResourceBundle rb) {
         borderPane.heightProperty().addListener(e -> contentController.updateScrollSize(borderPane));
     }
 
+    /**
+     * Initialize the toggle buttons Phasing, Attributes and Contents. On mac these buttons will appear right
+     * aligned in a separate top menu in the window, otherwise they will appear right aligned in the file menu.
+     */
+    public void initializeToggleButtons() {
+        MenuBar menuBar = AnalystApp.getMenuBar();
+        ToggleButton phaserButton = new ToggleButton("Phasing");
+        ToggleButton attributesButton = new ToggleButton("Attributes");
+        ToggleButton contentButton = new ToggleButton("Content");
+        attributesButton.setOnAction(e -> toggleSideBarAttributes(phaserButton, attributesButton, contentButton));
+        contentButton.setOnAction(e -> toggleSideBarAttributes(phaserButton, attributesButton, contentButton));
+        phaserButton.setOnAction(e -> toggleSideBarAttributes(phaserButton, attributesButton, contentButton));
+        phaserButton.getStyleClass().add("toolButton");
+        attributesButton.getStyleClass().add("toolButton");
+        contentButton.getStyleClass().add("toolButton");
+        SegmentedButton groupButton = new SegmentedButton(phaserButton, contentButton, attributesButton);
+        if (!AnalystApp.isMac()) {
```

Comment:
We could invert the if/else order to avoid the negation.

Reply:
done